51a.md

```diff
@@ -0,0 +1,18 @@
+NIP-51a
+======
+
+User Public Key List
+-------------------------
+
+`draft` `optional` `author:fiatjaf` `author:arcbtc` `author:monlovesmango` `author:eskema` 
+
+An event with kind `30000` is defined as a list event (see [NIP-51 - Lists](51.md)) for user public keys. The list of tags should follow the format of kind 3 events as defined in [NIP-02 - Contact List and Petnames](02.md).
+
+Commonly established list parameter names for this kind are:
+- `mute` - a list of tags that the user wants to mute
+- `follow` - a list of tags that the user wants to follow
+- `pin` - a list of tags that the user wants to pin (for display on profile)
+- `bookmark` - a list of tags that the user wants to bookmark
+
+Outside of the commonly established list parameter names above, users are allowed to create lists with custom names.
```

Comment:
This sentence is not needed. Of course they are allowed and they will do so at which point this nip will get amended to help interoperability.

```suggestion
```

Reply:
I think it is good to let people know it is ok to use names outside of those defined standardly. whats the harm?